pkg/skaffold/build/builder_mux_test.go

```diff
@@ -92,6 +92,71 @@ func TestNewBuilderMux(t *testing.T) {
 	}
 }
 
+func TestGetConcurrency(t *testing.T) {
+	tests := []struct {
+		description         string
+		pbs                 []PipelineBuilder
+		cliConcurrency      int
+		expectedConcurrency int
+	}{
+		{
+			description: "default concurrency - builder and cli concurrency unset.",
+			pbs: []PipelineBuilder{
+				&mockPipelineBuilder{nil, "local"},
+				&mockPipelineBuilder{nil, "gcb"},
```

Comment:
I'm thinking it might be worth explicitly specifying field names — it wasn't clear to me that "local" and "gcb" were the builder-type (go/go-style/decisions#literal-field-names).

Reply:
it's the default in IntelliJ. Will add the names.
![image](https://user-images.githubusercontent.com/1534539/158454761-047d1f92-a201-4341-bf5f-38526b6e5efa.png)